ci-operator/jobs/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-periodics.yaml

```diff
@@ -0,0 +1,53 @@
+#build target for this job has to be different, so that it could
+#pull the latest release.yaml and bundle into to operator image
+periodics:
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: "15m" #keeping for the testing purpose
+  name: openshift-tektoncd-pipeline-operator-nightly-tests
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --git-ref=openshift/tektoncd-pipeline-operator@master
```

Comment:
Please use `extra_refs` instead of this.
@stevekuznetsov Do we need to add a validatation for this?

Reply:
We also don't rehearse periodic jobs with no `extra_refs`. ref: [here](https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/4158/pull-ci-openshift-release-master-pj-rehearse/4706
)